util/libcrypto.num

```diff
@@ -4379,3 +4379,7 @@ BN_priv_rand_range                      4322	1_1_1	EXIST::FUNCTION:
 ASN1_TIME_normalize                     4323	1_1_1	EXIST::FUNCTION:
 ASN1_TIME_cmp_time_t                    4324	1_1_1	EXIST::FUNCTION:
 ASN1_TIME_compare                       4325	1_1_1	EXIST::FUNCTION:
+ossl_ctype_check                        4326	1_1_1	EXIST::FUNCTION:
+ossl_tolower                            4327	1_1_1	EXIST::FUNCTION:
+ossl_toupper                            4328	1_1_1	EXIST::FUNCTION:
+ossl_isalnum                            4329	1_1_1	EXIST::FUNCTION:
```

Comment:
Are they needed by libssl?  If not, then I completely agree.

If they are needed, however, then I suggest *one* ctype function which takes a flag byte saying what test to perform:
```
extern int ossl_ctype(unsigned char flag, unsigned char c);
```
And for cleverness, ```flag``` can be a char constant, like 'A' for isupper, etc.
(Use define's of course.  But that use *will* be portable.)